lib/simple_twitter.rb

```diff
@@ -23,13 +23,15 @@ def initialize(bearer_token: nil,
     %i[get post put delete].each do |m|
       class_eval <<~EOD
         # @return [Object] parsed json data
-        def #{m}(url, params={})
-          JSON.parse(#{m}_raw(url, params).to_s, symbolize_names: true)
+        def #{m}(url, params={}, json=nil)
```

Comment:
**[QUESTION]** If you don't mind making a breaking change, I think it would be easier for me to understand if it were a keyword argument.

e.g.

```ruby
def get(url, params: {}, json: nil)
```

What do you think?

Reply:
Or, I can add a method like the following

```ruby
def post_with_json(url, json = {})
```

Reply:
> If you don't mind making a breaking change, I think it would be easier for me to understand if it were a keyword argument.

Fine, I don't mind for a breaking change if it is noted in `CHANGELOG.md`.

Reply:
> Fine, I don't mind for a breaking change 

Thank you! I changed positional args to keyword args at c5c746a and 7da8da2